task/rpm-ostree/0.1/rpm-ostree.yaml

```diff
@@ -126,6 +126,25 @@ spec:
       else
         CONFIG_FILE_ARG="  --image-config=source/$CONFIG_FILE "
       fi
+
+      prefetched_rpms=./cachi2/output/deps/rpm/$(uname -m)
+      if [ -d "$prefetched_rpms" ]; then
+        # move all repo files out of the source repository to avoid conflicts with the cachi2.repo
+        mkdir /tmp/original-repos
+        find ./source -maxdepth 1 -name '*.repo' -exec mv {} /tmp/original-repos \;
+
+        # copy the platform-specific cachi2.repo into the source repository
+        cp "$prefetched_rpms/repos.d/cachi2.repo" ./source
```

Comment:
Just an FYI: This one will become `$prefetched_rpms/<arch>/repos.d/cachi2.repo`

Reply:
prefetched_rpms is the arch-specific directory - I will rename the variable